build-system/tasks/visual-diff/index.js

```diff
@@ -320,6 +322,31 @@ function createPercyPuppeteerController(assetGlobs) {
   }
 }
 
+/**
+ * Cleans up any existing AMP config from the runtime and 3p frame.
+ */
+function cleanupAmpConfig() {
+  log('verbose', 'Cleaning up existing AMP config');
+  AMP_RUNTIME_TARGET_FILES.forEach(targetFile => {
+    execOrDie(
+        `gulp prepend-global --local_dev --target ${targetFile} --remove`,
+        {'stdio': 'ignore'});
+  });
+}
+
+/**
+ * Applies the prod AMP config to the runtime and 3p frame with --fortesting.
+ */
+function applyAmpConfig() {
```

Comment:
This has a different purpose now, since `AMP_CONFIG` is now guaranteed to be applied during `gulp build`. For clarity, rename this to something like `enableTestMode`.

Reply:
Done